ramls/schemas/profiles/jobProfile.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema":"http://json-schema.org/draft-04/schema#",
+  "description":"Job profile",
+  "type":"object",
+  "additionalProperties":false,
+  "properties":{
+    "id":{
+      "description":"Unique identifier",
+      "type":"string",
+      "$ref":"../../raml-util/schemas/uuid.schema"
+    },
+    "name":{
+      "description":"Job Profile name",
+      "type":"string"
+    },
+    "destination":{
+      "description":"Final destination place where to send file",
+      "type":"string"
+    },
+    "description":{
+      "description":"Mapping Profile description",
+      "type":"string"
+    },
+    "userInfo":{
+      "description":"First name, last name and username of the user, who updated the Job Profile",
+      "type":"object",
+      "$ref":"userInfo.json"
+    },
+    "mappingProfileId":{
+      "description":"Associated mapping profile id",
+      "type":"string",
+      "$ref":"../../raml-util/schemas/uuid.schema"
+    },
+    "protocol":{
+      "description":"TCP/IP settings",
+      "type":"array",
+      "items":{
+        "type":"object",
+        "$ref":"protocol.json"
+      }
+    },
+    "metadata":{
+      "description":"Metadata provided by the server",
+      "type":"object",
+      "$ref":"../../raml-util/schemas/metadata.schema",
+      "readonly":true
+    }
+  },
+  "required":[
+    "id",
+    "destination"
```

Comment:
Why is destination a required field, for our current implementation we are not handling destination.